app/controllers/spaces_controller.rb

```diff
@@ -60,6 +60,10 @@ def space_params
       policy_scope(Space).friendly.find(params[:id])
     elsif params[:space]
       policy_scope(Space).new(space_params)
+    elsif params[:enforce_ssl] == 1
+      policy_scope(Space).friendly.find(params[:id]).update(enforce_ssl: true)
+    elsif params[:enforce_ssl] == 0
+      policy_scope(Space).friendly.find(params[:id]).update(enforce_ssl: false)
```

Comment:
Not sure if this is the correct way to update `Space` record based on the checkbox value.

Reply:
Hmm, this does seem a little off-the-rails to me. Normally, if `space_params` includes `enforce_ssl`  it will update just fine from the browser without needing to do a comparison check; so you shouldn't have to change the controller since you're permitting the new param in the `SpacePolicy`.